yarn-project/foundation/src/wasm/empty_wasi_sdk.ts

```diff
@@ -1,4 +1,4 @@
-import { createDebugOnlyLogger } from '../log/index.js';
+import { createLogger } from '../log/index.js';
 
```

Comment:
isnt this entirely unused?

Reply:
this is now defined in the C++

Reply:
for that matter, the whole wasm folder in foundation is not used

Reply:
I see. Nuked that whole dir as well in https://github.com/AztecProtocol/aztec-packages/pull/13187/commits/adf51d565d28614cea7096869b096a9ec47616ca